region/crateria/west/Gauntlet Energy Tank Room.json

```diff
@@ -705,7 +732,23 @@
       "link": [3, 4],
       "name": "Base",
       "requires": [
-        "h_canBombThings"
+        "h_canUsePowerBombs"
+      ]
+    },
+    {
+      "link": [3, 4],
+      "name": "Bombs",
+      "requires": [
+        "h_canUseMorphBombs",
+        {"or": [
+          "canTrickyJump",
```

Comment:
the two-cycle is what should be in logic, so I dont think it should require a tricky jump. I would think a careful jump or spring, but I would be ok with just careful.